opentok/Opentok-Python-SDK
# [ECO-2972] [Issue #121] OpenTok.start_archive() doesn't have parameter 'resolution'

Reopening of #123, fixes #121 .
**In This PR:**
- Added `resolution` parameter to `start_archive()` method
- Added Enums, helper methods, and surrounding logic to support the new parameter
- Updated README with details of the above
- Added Unit Tests for Above
- Update sample/Archiving/ web app to add resolution options

**How to Test**
- Check unit tests work
- Manually: Can run the sample app at sample/Archiving/archiving.py - I recommend printing `self.resolution` [here](https://github.com/maikthomas/Opentok-Python-SDK/blob/feature/archive_resolution/opentok/archives.py#L129) to check it goes all the way through the flow



See new resolution options on sample app ⬇️ 
![image](https://user-images.githubusercontent.com/16080485/42220602-bea0121a-7ecf-11e8-9d70-647b138f532f.png)
